HealthcareSystem/UserService/Model/Jmbg.cs

```diff
@@ -26,10 +27,11 @@ public override int GetHashCode()
 
         private void Validate()
         {
-            if (!IsRegular(Value))  throw new Exception("Invalid jmbg");       
+            if (string.IsNullOrEmpty(Value)) throw new ValidationException("Jmbg does not exist!");
```

Comment:
I wouldn't say 'doesn't exist' here, maybe something like 'Jmbg can't be empty' or 'Jmbg is required' is more accurate?